docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1178,7 +1198,10 @@ BESU_PRIVACY_PUBLIC_KEY_FILE=Orion/nodeKey.pub
 privacy-public-key-file="Orion/nodeKey.pub"
 ```
 
-Path to the [public key of the Orion node](../../Concepts/Privacy/Privacy-Overview.md#besu-and-orion-keys).     
+Path to the [public key of the Orion node](../../Concepts/Privacy/Privacy-Overview.md#besu-and-orion-keys). 
+
+!!! important
+    Cannot be used when [`--privacy-multi-tenancy-enabled`](#privacy-multi-tenancy-enabled) is `true` 
```

Comment:
Is there a sensible error message if you do try to use both at once?

Reply:
Yes, it's `Privacy multi-tenancy and privacy public key cannot be used together`